doc/ssl/SSL_CTX_use_serverinfo.pod

```diff
@@ -30,6 +30,11 @@ must consist of a 2-byte Extension Type, a 2-byte length, and then length
 bytes of extension_data.  Each PEM extension name must begin with the phrase
 "BEGIN SERVERINFO FOR ".
 
+If more than one certificate (RSA/DSA) is installed using
+SSL_CTX_use_certificate(), the serverinfo extension will be loaded into the
+last certificate installed.  If e.g. the last item was a RSA certificate, the
+loaded serverinfo extension data will be loaded for that certificate.
```

Comment:
Can we mention that if you want it for multiple certificates it can be called multiple times - once after each time a cert is loaded?